resources/views/components/threads/overview-summary.blade.php

```diff
@@ -49,13 +49,13 @@
         <div class="flex gap-x-5">
             <span class="flex items-center gap-x-2">
                 <x-heroicon-o-thumb-up class="w-6 h-6" />
-                <span>{{ count($thread->likes()) }}</span>
+                <span>{{ $thread->like_count }}</span>
                 <span class="sr-only">Likes</span>
             </span>
 
             <span class="flex items-center gap-x-2">
                 <x-heroicon-o-chat-alt-2 class="w-6 h-6" />
-                <span>{{ count($thread->replies()) }}</span>
+                <span>{{ $thread->reply_count }}</span>
```

Comment:
Use methods for these counts.

Reply:
They are directly returned from the query using `withCount` to prevent the need for additional queries

Reply:
Can't we wrap them in `$thread->replyCount()`?

Reply:
Or nvm, this is fine for now.